### ISSUE TITLE
Enhancement - Track cost of energy used

### ISSUE THREAD

**Issue by paul-thomas1964** [xoseperez/node-red-contrib-power-monitor#20]
Could energy cost also be tracked alongside kw/h?
This cost per unit could be set on the node, or set via a parameter on an incoming message.  This could dynamically change if using a split tier tariff to provide the correct cost over the period.

**Comment by hjuhlin** [comment]
Yes! This is very needed and I started to work with that localy yesterday to see how hard it was to do. Looks very easy, so I should be done soon :)
